build.gradle

```diff
@@ -221,13 +265,14 @@ configure(subprojects.findAll { !it.name.contains("android") }) {
     }
 
     copyAsciiDoc.finalizedBy(asciidoctor)
+
+    project.ext.publishVariant('java')
```

Comment:
I still find it a bit confusing to use `ext` here. 
But haven't a better idea now.